tests/test_gpu_openmp.py

```diff
@@ -314,12 +335,14 @@ def iso_acoustic(self, **opt_options):
 
         assert np.isclose(norm(rec), 490.55, atol=1e-2, rtol=0)
 
-    @pytest.mark.parametrize('opt_options', [
-        {},
-        {'linearize': True},
+    @pytest.mark.parametrize('opt', [
+        ('advanced', {}),
+        ('advanced', {'linearize': True}),
+        ('blocking', ),
```

Comment:
this one is probably overkill since blocking is already tested below in tandem with linearize